### DISCUSSION TITLE
Relation Expand from Parent to Child

### DISCUSSION THREAD

**Discussion by hungcrush** [pocketbase/pocketbase#1556] [Q&A] [answered]
I have a question related Relation.

<img width="675" alt="CleanShot 2023-01-10 at 17 15 47@2x" src="https://user-images.githubusercontent.com/8098916/211524016-4e9fa0b3-11e4-43d3-b2eb-2731abfcdc09.png">

Can get get `posts` related when I getting categories list ? (like expand)

**Comment by ganigeorgiev** [comment, marked as answer]
If I'm understanding correctly your structure, I think you should be able to do that with indirect expand - https://pocketbase.io/docs/expanding-relations/#indirect-expand, aka. something like:

```js
const result = await pb.collection('categories').getList(1, 30, {
   expand: 'posts(category)' // "posts" is the indirect collection name and "category" is the relation field name
})
```

**Reply by hungcrush** [reply]
Sorry for my english.
Thank you so much. Why you can do that?? 😭

**Reply by HiImJulien** [reply]
**Note:** This still work as of 2024. However, if there is nothing to join on, the attribute `expand` is left `undefined`.
